WebRoot/js/zimbraMail/calendar/model/ZmAppt.js

```diff
@@ -252,6 +252,42 @@ function(controller, callback) {
 	this.apptClone.getDetails(null, respCallback);
 };
 
+ZmAppt.prototype._getItemAttributes =
+function (hashMap, itemType) {
+		var ptstStatus = {};
+		var statusAttendees;
+		var attendeesText;
+		var hideAttendees = true;
+
+		statusAttendees = ptstStatus[ZmMsg.ptstAccept] = this._getPtstStatus(hashMap, ZmCalBaseItem.PSTATUS_ACCEPT);
+		hideAttendees = hideAttendees && !statusAttendees.count;
```

Comment:
do we need to define statusAttendees ? we can use ptstStatus[ZmMsg.ptstAccept].count instead

Reply:
removed